Source/Core/Core/PowerPC/Jit64/Jit_Branch.cpp

```diff
@@ -160,7 +164,8 @@ void Jit64::bx(UGeckoInstruction inst)
 
   if (IsDebuggingEnabled())
   {
-    WriteBranchWatch<true>(js.compilerPC, js.op->branchTo, inst, RSCRATCH, RSCRATCH2, {});
+    static_assert((BitSet32{ABI_PARAM1} & ABI_ALL_CALLER_SAVED) != BitSet32{});
```

Comment:
What is the purpose of this static assert? Maybe I'm missing something, but I don't see why it would matter if ABI_PARAM1 is caller-saved.

Reply:
I have probably just misunderstood how to document that this optimization is only safe after a GPR flush. I can replace these with comments saying this more directly?

Reply:
Okay, I understand then. The static assert didn't actually check whether the GPRs had been flushed – that would have required a runtime assert. The new comment looks fine.